lib/blockchain/chain.js

```diff
@@ -1686,6 +1686,16 @@ class Chain extends AsyncEmitter {
    */
 
```

Comment:
Checkpoints default to `true` which is particularly important for the start of this chain to prevent the chain width expansion attack. :+1:

Reply:
Yeah, this rejection below should be the default in the future.